cookbooks/aws-parallelcluster-platform/attributes/platform.rb

```diff
@@ -9,6 +9,10 @@
 # ArmPL
 default['conditions']['arm_pl_supported'] = arm_instance?
 
+# Enroot + Pyxis
+default['cluster']['enroot']['version'] = '3.4.1'
+default['cluster']['pyxis']['version'] = 'v0.19.0'
```

Comment:
Does the `v` need to be in the version value?

Just going to keep a string of numbers similar to what we do should be enough